src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,33 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+
+        StringBuilder lines = new StringBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.append(line);
+            }
```

Comment:
you can find a way to iterate over each character already at this point
then, using a method in class Character, you can check if the specific char suits you
and add it to the set